myproject/templates/single_entry.jinja2

```diff
@@ -0,0 +1,13 @@
+{% extends "layout.jinja2" %}
+{% block body %}
+
+    <body>
+        <h1>#{{ id }} - {{ title }}</h1>
+        {{ creation_date}}
+        <p>
```

Comment:
Eventually, you may wish to put HTML into your entry.  In which case it might contains `<p>` tags of its own.  I would suggest that you use a more neutral HTML container for the body of your journal entry.  Perhaps a `<div>` tag instead.